test/presets/bem.js

```diff
@@ -151,6 +151,43 @@ doAsync(function() {
 
 // Comments with url can have maximum line length > 120 ................... https://github.com/yandex/codestyle/blob/master/js.md
 
+// `i-bem` should be named BEM
+// `i-bem__dom` should be named BEMDOM
+modules.define('something-truly', ['i-bem__dom', 'something-else'], function(provide, BEMDOM, SomethingElse) {
+
+// Should use `this.name` if possible for BEMDOM.decl
+// Should have @lends jsdoc tag
+// Should use provide right in place with declaration if possible
+provide(BEMDOM.decl(this.name, /** @lends something-truly.prototype */ {
+    // Modifier names and values should have single quotes
+    beforeSetMod : {
+        'hovered' : {
+            'true' : function() {
+            }
+        }
+    },
+
+    // Should be line-break between mods declarations
+    onSetMod : {
+        // `any` (`*`) value should be used explicitly
```

Comment:
Where this requirement is came from? Generally we use `*` only in case `other` mod handlers are present. [Here is some random example from `menu`](https://github.com/bem/bem-components/blob/e594e82f3a59f7a52f97b28e2363b12740e9e6fa/common.blocks/menu/menu.js#L36)